index.html

```diff
@@ -14,6 +14,59 @@
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    <header class="header">
+        
+        <img src="./img\karma-logo.svg" alt="karma-logo" id="logo"/>
+        <nav class="navbar">
+            <ul class="list-nav">
+                <li><b><a href="#">Meet Karma</a> </b></li>
+                <li><a href="#">How it work</a></li>
+                <li><a href="#">Store</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="#">Help</a></li>
+                <li><a href="#">Login</a></li>
+            </ul>
+        </nav>
+        <img src="img\menu-hamburger.svg" alt="menu-hamburger" id="menu">
+     
+    
+</header>
+<main>
+    <section class="section-1">
+        <h2> Introducing Karma</h2>
+        <p> Bring WiFi with you, everywhere you go.</p>
+        <button> Learn More</button>
+    </section>
+    <section class="section-2">
+         <h2 class="h-font">Everyone Needs a little Karma</h2>
+         <div class="section3">
+             <div>
+                 <img src="\img\icon-devices.svg" alt="icon-devices"/>
+                 <p>Internet or all adevices</p>
+             </div>
+             <div>
+                <img src="\img\icon-coffee.svg" alt=""/>
```

Comment:
```suggestion
                <img src="\img\icon-coffee.svg" alt="Coffee cup"/>
```
We should give these images `alt` descriptions, with a description of the image itself